ulimit_unix_test.go

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2019, Daniel Martí <mvdan@mvdan.cc>
```

Comment:
2022?

Reply:
I thought about it but I got lazy since it doesn't really matter. Then I went on to type these two sentences which is significantly more effort :wink: Fixed.